src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -529,7 +529,14 @@ async fn healthcheck(
                     Err(HealthcheckError::GroupNameError.into())
                 }
             }
-            None => Err(HealthcheckError::NoLogGroup.into()),
+            None => {
+                if config.create_missing_group.unwrap() == true {
```

Comment:
I wasn't sure about that, because at the time of healthchecking the config should never be `None` - since the config as you say would default to true if unset

Reply:
It could be `None` there still. The values are defaulted when the `CloudwatchLogsSvc` struct is created: 

https://github.com/timberio/vector/blob/4d1ea4f336a0d7b97aa8e82fee55180ae063f8fe/src/sinks/aws_cloudwatch_logs/mod.rs#L281-L300

This healthcheck function just gets the raw config though.

Reply:
Ah, so at this point the values could be Some(true), Some(false), or None?

Reply:
Yep! That's right.